src/lib/utils/translations.ts

```diff
@@ -0,0 +1,60 @@
+import { Languages } from '../types';
```

Comment:
You can `import` json files here. So you can place translations to something like `languages/cs.json` and then `import cs from 'languages/cs.json'` and use as `cs[message]`. It is more flexible for extension.

Reply:
Nice, we may introduce something like this in the future, if there will ever be more translations needed. But for now I don't think we need this :).